File: ARCADE/The Core/Level 2_6: tennisSet.cs
    using System;
     
    public class Test
    {
    	public static bool tennisSet(int score1, int score2){
            if ((Math.Max(score1, score2) == 6 && Math.Max(score1, score2) - Math.Min(score1, score2) >= 2) || (Math.Max(score1, score2) == 7 && Math.Min(score1, score2) >= 5 && Math.Min(score1, score2) < 7)) return true;
            return false;
        }
    	public static void Main()
    	{
    		int score1 = 3, score2 = 6;
    		Console.WriteLine(tennisSet(score1, score2));
    		int score3 = 8, score4 = 5;
    		Console.WriteLine(tennisSet(score3, score4));
    	}
    }

File: ARCADE/The Core/Level 2_6: tennisSet.pas
program tennisSet;
uses Math;
function tennisSet(score1: integer; score2: integer): boolean;
begin
    if(((Max(score1,score2) = 6) and (Max(score1,score2) - Min(score1,score2) >= 2)) or
    (((Max(score1,score2) = 7) and (Min(score1,score2) >= 5) and (Min(score1,score2) < 7)))) 
    then tennisSet := true
    else tennisSet := false;
end;

var score1, score2 : integer;
begin
	score1 := 3; score2 := 6;
	writeln(tennisSet(score1, score2));
	score1 := 8; score2 := 5;
	writeln(tennisSet(score1, score2));
	score1 := 6; score2 := 5;
	writeln(tennisSet(score1, score2));
end.
